builder/vmware/iso/config.go

```diff
@@ -177,15 +126,6 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		}
 	}
 
-	if c.DiskTypeId == "" {
```

Comment:
This default value is being removed from the vmware-iso. Is it intended?  According to the docs, I would guess this shouldn't be removed.